src/Common/src/System/Net/Http/HttpHandlerDefaults.cs

```diff
@@ -14,6 +14,7 @@ internal static class HttpHandlerDefaults
         public const int DefaultMaxAutomaticRedirections = 50;
         public const int DefaultMaxConnectionsPerServer = int.MaxValue;
         public const int DefaultMaxResponseDrainSize = 1024 * 1024;
+        public static readonly TimeSpan DefaultMaxResponseDrainTime = TimeSpan.FromSeconds(5);
```

Comment:
HttpWebRequest in netfx appears to use 500ms by default, but allows overriding via an appconfig setting.  500ms seems too short to me, but we can make this whatever we think is most appropriate.

Reply:
5 sec does seem like a lot.  Maybe 2 or 3 sec?

Reply:
Sure, I'll make it 2.